internal/catalogmetadata/types.go

```diff
@@ -11,6 +11,12 @@ import (
 	"github.com/operator-framework/operator-registry/alpha/property"
 )
 
+const (
+	MediaTypePlain          = "plain+v0"
+	MediaTypeRegistry       = "registry+v1"
+	PropertyBundleMediaType = "olm.bundle.mediatype"
+)
```

Comment:
I was a bit confused since this (and `PackageRequired` an some other bits in this file) is not related to GVK removal. But we still need these changes for further entity source removal.

I'm ok to merge it like this.